Core/Blob/Extensions/BlobServiceBuilderExtensions.cs

```diff
@@ -38,12 +39,37 @@ public static class BlobServiceBuilderExtensions
 
             builder
                 .AddProgramInitializer<BlobInitializerService>()
-                .AddAssemblies(typeof(AutoMapperProfile).Assembly)
+                .AddAssemblies(typeof(Data.AutoMapperProfile).Assembly)
                 .AddDatabase<BlobDbContext>(connectionStringName, connectionStringPassword)
+                .AddAuditTrailing()
                 .Services
                 .AddScoped<IBlobManagementService, BlobManagementService>();
 
             return builder;
         }
+
+        public static IApplicationSetupBuilder AddBlobService(this IApplicationSetupBuilder builder,
```

Comment:
It's nice you add another method for instantiating the blob service with the provider coming from settings in order to keep the compatibility with the other projects (actually Allie project :-) )
I'm taking into account of using the provider from settings in our apps, too. Then, I can merge everything is one method.